projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics-loader.service.spec.ts

```diff
@@ -16,25 +47,10 @@ class CustomQualtricsLoaderService extends QualtricsLoaderService {
   }
 }
 
-const mockQsi = {
-  API: {
-    unload: (): void => {},
-    load: () => {
-      return {
-        done: (_intercept: Function) => {},
-      };
-    },
-    run: (): void => {},
-  },
-};
-
-fdescribe('QualtricsLoaderService', () => {
+describe('QualtricsLoaderService', () => {
+  let mockedWindowRef;
   let service: QualtricsLoaderService;
   let winRef: WindowRef;
-  const eventListener: Record<'qsi_js_loaded', Function> = <
-    Record<'qsi_js_loaded', Function>
-  >{};
-  let mockedWindowRef;
 
   beforeEach(() => {
     mockedWindowRef = {
```

Comment:
Let's turn `mockedWindowRef` into a const here, and remove the `let mockedWindowRef;` from line 51?